x-pack/plugins/cloud_security_posture/server/create_indices/create_indices.ts

```diff
@@ -197,6 +197,13 @@ const updateIndexTemplate = async (
 ) => {
   const { indexTemplateName, indexPattern, template, composedOf, _meta } = indexTemplateParams;
   try {
+    // for now, remove from object so as not to update stream or data stream properties of the index until type and name
```

Comment:
In the examples you brought ([here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/index_management/server/routes/api/data_streams/register_post_route.ts#L46-L53) and [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts#L674:L681)) the deletion happens before `putMapping` without effecting the actual index template

Can you elaborate why here we are changing the index template

Reply:
besides that, the stacktrace also shows the failing request is `putMappings`

Reply:
@kfirpeled my fault, I switched branched and pushed the wrong one, you deferentially right.